python/singa/net.py

```diff
@@ -471,8 +471,8 @@ def save(self, f, buffer_size=10, use_pickle=False):
             if f.endswith('.bin'):
                 f = f[0:-4]
             sp = snapshot.Snapshot(f, True, buffer_size)
-            v = tensor.from_numpy(np.array([__version__]))
-            sp.write('SINGA_VERSION', v)
+            #v = tensor.from_numpy(np.array([__version__]))
```

Comment:
the two lines should not be commented out

Reply:
maybe just write version information to .desc file is sufficient. if use sp.write('singa_version'，v) to write, then the useful information just be stored in .bin file. It brings unnecessary complexity to decode. And for the corresponding .desc file, it should be 'parameter name: SINGA_VERSION	data type: 0	dim: 1	shape: 1', which are all the useless information.

Reply:
I rethink about it. Store all useful information in one particular file may be better, which mean store all information just in .bin file. this is the file computer to read. it requires to write v tensor. 
computer no need to read .desc file, it can be designed just for user to read. 
In former implementation, computer read .desc file just to do version check, which can be simplified.

Reply:
it's fine to remove the version check. the desc file is human readable, which is convenient for checking the meta info of the .bin file.